platooning_strategic_IHP/src/platoon_strategic_ihp.cpp

```diff
@@ -659,18 +825,21 @@ namespace platoon_strategic_ihp
             // read ecef location from strategy params.
             cav_msgs::LocationECEF ecef_loc;
             ecef_loc = mob_op_find_ecef_from_INFO_params(strategyParams);
-
-            // use ecef_loc to calculate front dtd in m.
+            
+            // use ecef_loc to calculate front Dtd in m.
             lanelet::BasicPoint2d incoming_pose = ecef_to_map_point(ecef_loc);
             double frontVehicleDtd = wm_->routeTrackPos(incoming_pose).downtrack;
-            
+
+            // use ecef_loc to calculate front Ctd in m.
+            double frontVehicleCtd = wm_->routeTrackPos(incoming_pose).crosstrack;
+
             // Use pm_ to find platoon end vehicle and its downtrack in m.
             int rearVehicleIndex = pm_.getTotalPlatooningSize()- 1;
             double rearVehicleDtd = pm_.platoon[rearVehicleIndex].vehiclePosition; 
             
             // downtrack of the platoon leader --> used for frontal join
```

Comment:
This comment doesn't match the compilable code. One is talking about leader, one is talking about rear vehicle.